.gitignore

```diff
@@ -0,0 +1,62 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I think we should scope the exclusions here to what we actually need. E.g. Just:

```
__pycache__/
*.py[cod]
venv/
build/
.eggs/
```

we can add more as we need them.